src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,109 @@
+package com.notably.logic.suggestion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.commons.path.AbsolutePath;
+import com.notably.model.Model;
+import com.notably.model.ModelManager;
+import com.notably.model.block.BlockModel;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+import com.notably.model.suggestion.SuggestionModel;
+import com.notably.model.suggestion.SuggestionModelImpl;
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+import com.notably.testutil.TypicalBlockModel;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    private static Model model;
+
+    public static Model getModel() {
+        BlockModel blockModel = TypicalBlockModel.getTypicalBlockModel();
+        SuggestionModel suggestionModel = new SuggestionModelImpl();
+        ViewStateModel viewStateModel = new ViewStateModelImpl();
+        model = new ModelManager(blockModel, suggestionModel, viewStateModel);
+
+        return model;
+    }
+
+    public static AbsolutePath getToCs2103t() {
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSuggestionsToCs2103t() {
```

Comment:
To be honest, this name sort of confused me, since "suggestions to CS2103T" does not ring any bell for me. How about we call it "getSuggestionsForCs2103tPathInput()`?